### PR TITLE
feat(naming): dynamically generate unique name for auro-dropdown #127

### PR DESCRIPTION
**Pull request by jason-capsule42**
# Alaska Airlines Pull Request

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

**Fixes:** #127

## Summary:

This work shows using a dynamically generated hash to create a unique element name used during the custom element definition

## Type of change:

Please delete options that are not relevant.

- [x] New capability
- [ ] Revision of an existing capability
- [ ] Infrastructure change (automation, etc.)
- [ ] Other (please elaborate)

## Checklist:

- [x] My update follows the CONTRIBUTING guidelines of this project
- [x] I have performed a self-review of my own update

**By submitting this Pull Request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

_Pull Requests will be evaluated by their quality of update and whether it is consistent with the goals and values of this project. Any submission is to be considered a conversation between the submitter and the maintainers of this project and may require changes to your submission._

**Thank you for your submission!**<br>
-- Auro Design System Team

### CODE REVIEW

File: .github/workflows/testPublish.yml [hunk 1]
```diff
@@ -39,6 +39,7 @@ jobs:
         with:
           node-version: 16.x
       - run: npm ci
+      - run: npm run version
```
Comment (Izook) [inline .github/workflows/testPublish.yml:42]: should this be `build:version`?

Comment (blackfalcon) [reply .github/workflows/testPublish.yml:42]:
```
./combobox (jbaker/hashNamingPOC=) $ npm run version
npm ERR! Missing script: "version"
npm ERR!
npm ERR! To see a list of scripts, run:
npm ERR!   npm run

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/dalesande/.npm/_logs/2023-06-02T18_09_10_609Z-debug-0.log
```

**Comment by blackfalcon** [comment]
Please update commit https://github.com/AlaskaAirlines/auro-combobox/pull/128/commits/a7bf2174e0edfdabee8b5531f0dfd8eae4354383 to have notes that this repo was updated to Lit2.0 as well as create the new feature. 

Also commits https://github.com/AlaskaAirlines/auro-combobox/pull/128/commits/b8abdee6dfbdf79eb383e8d3a43e07814ae26953 and https://github.com/AlaskaAirlines/auro-combobox/pull/128/commits/4a130c80e0513d1b2858f24c9bf16833fa154448 can all be merged into the single https://github.com/AlaskaAirlines/auro-combobox/pull/128/commits/a7bf2174e0edfdabee8b5531f0dfd8eae4354383 commit. Those updates and a repeated `feat(naming): improving dynamic naming of deps` message has no real long-term historical value.

**Review by blackfalcon — DISMISSED** [review]
Please see other comments.

File: scripts/version.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Alaska Air. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+// import { AuroDependencyVersioning } from "./dependencyVersioning.mjs";
```
Comment (jason-capsule42) [inline scripts/version.js:6]: Remove this commented out line

File: src/style.scss [hunk 3]
```diff
@@ -5,7 +5,7 @@
 
 @import './../node_modules/@aurodesignsystem/webcorestylesheets/dist/utilityClasses/displayProperties';
 
-combobox-input {
+[auro-input] {
```
Comment (jason-capsule42) [inline src/style.scss:8]: Double check this?

**Review by blackfalcon — CHANGES_REQUESTED** [review]
Broken test https://github.com/AlaskaAirlines/auro-combobox/actions/runs/5292340572/jobs/9579105342

**Comment by jason-capsule42** [comment]
Closing pull request in favor of new attempt: https://github.com/AlaskaAirlines/auro-combobox/pull/148

**Pull request closed without merging**